examples/StereoDepth/rgb_depth_aligned.py

```diff
@@ -65,6 +82,15 @@
     frameRgb = None
     frameDepth = None
 
+    # Configure windows; trackbar adjusts blending ratio of rgb/depth
+    rgbWindowName = "rgb"
+    depthWindowName = "depth"
+    blendedWindowName = "rgb-depth"
+    cv2.namedWindow(rgbWindowName, cv2.WINDOW_AUTOSIZE)
```

Comment:
We prefer not to set window properties, as it might react "weird" on different hosts (e.g. RPI).
In this case it's fine because `WINDOW_AUTOSIZE` is the default anyway.
If you make further changes I would suggest removing, so it matches the rest of the examples.

Reply:
Ah, that makes sense. Removed.